lib/hubspot-ruby.rb

```diff
@@ -28,3 +28,6 @@ def self.configure(config={})
 
   require 'hubspot/railtie' if defined?(Rails)
 end
+
+# Alias the module for those looking to use the stylized name HubSpot
+HubSpot = Hubspot
```

Comment:
I like this 👍 as I would also expect `HubSpot` instead of `Hubspot`. Shall we make a ticket to replace `Hubspot` with `HubSpot` as part of a `v1` release?

Reply:
Actually performing the swap touches a _bunch_ of code locations (see #128) for only a minor syntax change. Changing this would also mean that anyone who has been using this gem in their own projects would have to change every instance of `Hubspot` to `HubSpot`. This change means that either will work and I don't think it's too ambiguous about what module you're trying to reference. There is precedence for supporting these types of things, for example RestClient allows you to import it using `require "restclient"`, `require "rest-client"`, or `require "rest_client"` (https://github.com/rest-client/rest-client/tree/master/lib)

Reply:
Neat, I didn't know that about `rest-client`. I should clarify that I'm on board with making this change now so users can choose between `HubSpot` and `Hubspot`.

I would also be on board with making the larger change of swapping out `Hubspot` to `HubSpot`, even though the change would touch a large number of files. To help users continue to upgrade without having to replaces their existing code to use the new `HubSpot` namespace, we could replace this assignment with the inverse: `Hubspot = HubSpot`.

I don't feel strongly about it :) I Just figured I'd bring it up as an option to help promote consistency and eventually remove the need for this "alias".

Reply:
Sounds good. I don't feel strongly about it either way but I figure a change that requires less work from downstream users is probably preferable.